lib/power_action_utils.pm

```diff
@@ -81,12 +81,19 @@ sub reboot_x11 {
     my ($self) = @_;
     wait_still_screen;
     if (check_var('DESKTOP', 'gnome')) {
-        if (is_tumbleweed) {
+        # Get gnome version
+        x11_start_program('xterm -geometry 150x45+5+5', target_match => 'xterm');
+        become_root;
+        my $gnome_version = script_output("rpm -q --queryformat '%{VERSION}' gnome-shell");
+        type_string "killall xterm\n";
+
+        # Change in reboot process was introduced with gnome 3.38.1
+        if (package_version_cmp($gnome_version, '3.38.1') < 0) {
```

Comment:
The change happened with gnome 3.38.0 (well, actually 3.37.x, but if you go 3.38.0 it's probably closest to reality/relevance to products)

Reply:
Thanks!